CHANGELOG.md

```diff
@@ -1,4 +1,6 @@
 ## Unreleased
+FEATURES:
```

Comment:
I think this is more of an improvement than a feature. Suggest to change this to `IMPROVEMENTS:`.

Reply:
Makes sense, will update.